operators/churner/churner.go

```diff
@@ -21,7 +21,7 @@ import (
 
 var (
 	bipMultiplier     = big.NewInt(10000)
-	secondsTillExpiry = 90 * time.Second
+	secondsTillExpiry = 3600 * time.Second
```

Comment:
Why extend to 3600s?

Reply:
90sec is too stringent for some operators to opt in, especially smart contract operator. though it is already merged from #481